csfieldguide/static/interactives/matrix-simplifier/js/matrix-simplifier.js

```diff
@@ -640,3 +680,25 @@ function validateInput() {
     $('.add-from-input').prop('disabled', false);
   }
 }
+
+
+/** Removes all equations and re-calculates output */
+function removeAllMatrices() {
+  $(".matrix").parent().remove();
+  currentMatricesOrder = [];
+  matricesStringFormat = [];
+  $('#remove-all-matrices').attr('disabled', true);
```

Comment:
Removing the last matrix manually doesn't disable the button
There is no adverse affect to this though
![image](https://user-images.githubusercontent.com/25916475/67050434-02265b00-f195-11e9-8210-de3029889922.png)